<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>940</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>940</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>940</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 21, 25);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>602</y>
      <width>464</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>17</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>17</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>IP адресс сервера</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_port1IP">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 5;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>17</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>17</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>UDP порт сервера</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_port1Port">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 5;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>17</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>17</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>IP адрес клиента</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_port2IP">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 5;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>17</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>17</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>UDP порт клиента</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_port2Port">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 5;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>382</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>34</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter SemiBold</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Настройки сетевого подключения</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>69</width>
           <height>22</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>69</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter SemiBold</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Статус:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lb_port1Status">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>69</width>
           <height>22</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>69</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter SemiBold</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 87, 87);</string>
         </property>
         <property name="text">
          <string>Закрыт</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>462</width>
      <height>437</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="trainStatus">
       <property name="minimumSize">
        <size>
         <width>460</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>460</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter SemiBold</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border: 2px solid white;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>ПОЕЗДА НЕТ НА УЧАСТКЕ ДКУ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorIndicate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>460</width>
         <height>360</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>460</width>
         <height>360</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 30px;
background-image: url(:/new/icons/Icons/No_train.svg);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>810</y>
      <width>462</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pB_port1Open">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter SemiBold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(117, 126, 207);
border-radius: 5;
}
QPushButton:hover{
background-color: rgb(209, 214, 255);
color: rgb(20, 20, 20);
border-radius: 5;
}</string>
       </property>
       <property name="text">
        <string>Открыть сокет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_port1Close">
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>225</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Inter SemiBold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(117, 126, 207);
border-radius: 5;
}
QPushButton:hover{
background-color: rgb(209, 214, 255);
color: rgb(20, 20, 20);
border-radius: 5;
}</string>
       </property>
       <property name="text">
        <string>Закрыть сокет</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
